content/en/monitors/guide/division-in-monitor-evaluations.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Division in Monitor Evaluations
+kind: guide
+aliases:
+  - /monitors/faq/Division-in-Monitor-Evaluations
+---
+
+## Overview
+
+Creating an alert based on a query with division is a common practice. There are some tools and behaviors that should be considered to ensure monitor settings are appropriate for evaluating these queries as intended.
+
+## Error Rate Example
+
+Suppose you want to monitor an error rate over 5 minutes using the metrics, `requests.error` and `requests.total`. There are 2 possible ways to perform this calculation.
+
+Consider a single evaluation performed with these aligned timeseries points for the 5 min timeframe:
+
+**Numerator**: `sum:requests.error{*}`
+
+```
+| Timestamp             | Value       |
+| :-------------------- | :---------- |
+| 2018-03-13 11:00:30   | 1           |
+| 2018-03-13 11:01:30   | 2           |
+| 2018-03-13 11:02:40   | 3           |
+| 2018-03-13 11:03:30   | 4           |
+| 2018-03-13 11:04:40   | 5           |
+```
+
+**Denominator**: `sum:requests.total{*}`
+
+```
+| Timestamp             | Value    |
+| :-------------------- | :------- |
+| 2018-03-13 11:00:30   | 5        |
+| 2018-03-13 11:01:30   | 5        |
+| 2018-03-13 11:02:40   | 5        |
+| 2018-03-13 11:03:30   | 5        |
+| 2018-03-13 11:04:40   | 5        |
+```
+
+### Division with `as_count()`
+
+Monitors involving division and at least 1 `as_count()` modifier use a separate evaluation path than other monitors. This impacts queries of the format `queryA.as_count() / queryB.as_count()`. Queries without division (`+`, `-`, or `*`) are not impacted.
```

Comment:
The current logic only checks whether the query has `as_count()`, so it affects queries with multiplication too. Evaluation order doesn't matter for addition and subtraction unless functions are applied (you get the same result regardless of the order).

Reply:
Changing this to just be more general.  I pulled this more or less from the old doc.